README.md

```diff
@@ -1,17 +1,21 @@
 ![BLS Wallet](./docs/images/bls-github-banner.svg)
 
-An Ethereum Layer 2 smart contract wallet that uses [BLS signatures](https://en.wikipedia.org/wiki/BLS_digital_signature) and aggregated transactions to reduce gas costs.
+## What is BLS Wallet?
 
-You can watch a full end-to-end demo of the project [here](https://www.youtube.com/watch?v=MOQ3sCLP56g)
+A set of components to easily utilize Ethereum Layer 2 smart contract wallets that use [BLS signatures](https://en.wikipedia.org/wiki/BLS_digital_signature) to aggregate transactions, in order to reduce gas costs. Our smart contract wallet supports recovery, atomic multi-action transactions, sponsored transactions and user-controlled upgradeability.
```

Comment:
Edited the intro here to give readers better context on what BLS Wallet is. Let me know if any disagreement/adjustments need making

Reply:
Suggestion: reverse the components you mention, namely putting 'reduce gas costs' first since that is the "why". The "How" is via bls sig aggregation on evm rollups.

Reply:
Some options with gas mentioned earlier in the sentence:

"A set of components to easily utilize Ethereum Layer 2 smart contract wallets, that reduce use gas costs by using BLS signatures to aggregate transactions."

"A set of components to easily utilize smart contract wallets that reduce gas costs on Ethereum Layers 2s, by using BLS signatures to aggregate transactions."

Reply:
eg. "A set of components to bring lower gas costs to evm rollups via ..."

Reply:
Thoughts on where to put the part about/how to mention SCWs? Few options in full:

1. A set of components to bring lower gas costs to EVM rollups via BLS signatures to aggregate transactions. Our smart contract wallet supports recovery, atomic multi-action transactions, sponsored transactions and user-controlled upgradeability.

2. A set of components to reduce gas costs on EVM Optimistic rollups by using BLS signatures to aggregate transactions. Our smart contract wallet supports recovery, atomic multi-action transactions, sponsored transactions and user-controlled upgradeability.

3. A set of components to reduce gas costs on EVM Optimistic rollups by using BLS signatures to aggregate transactions. This includes a smart contract wallet that supports recovery, atomic multi-action transactions, sponsored transactions and user-controlled upgradeability.

Reply:
Updated cheers, thanks for catching the "upgradability" typo as well.